packages/Search/src/details/DTK_DetailsDistributedSearchTreeImpl.hpp

```diff
@@ -106,26 +108,86 @@ struct DistributedSearchTreeImpl
 template <typename DeviceType>
 double DistributedSearchTreeImpl<DeviceType>::epsilon = 1.0e-6;
 
+template <typename View>
+inline Kokkos::View<typename View::traits::data_type, Kokkos::LayoutRight,
+                    typename View::traits::host_mirror_space>
+create_layout_right_mirror_view(
+    View const &src,
+    typename std::enable_if<!(
+        ( std::is_same<typename View::traits::array_layout,
+                       Kokkos::LayoutRight>::value ||
+          View::rank == 1 && !std::is_same<typename View::traits::array_layout,
+                                           Kokkos::LayoutStride>::value ) &&
+        std::is_same<typename View::traits::memory_space,
+                     typename View::traits::host_mirror_space::memory_space>::
+            value )>::type * = 0 )
+{
+    return Kokkos::View<typename View::traits::non_const_data_type,
```

Comment:
I think that you forgot to change this line. You cast away the const but it doesn't match the return type of the function

Reply:
Good catch.  Fixed.